CHANGELOG.md

```diff
@@ -0,0 +1,39 @@
+# Changelog for Palaver ZNC Module
+
+## Master
```

Comment:
Might be a bit confusing, as master is "also" 1.0.1 according to the readme etc. Maybe make master 1.0.2 :)